vyper/functions/functions.py

```diff
@@ -1225,12 +1225,11 @@ def sqrt(expr, args, kwargs, context):
     )
 
 
-def _clear():
-    raise CompilerPanic(
-        "This function should never be called! `clear()` is currently handled "
-        "differently than other functions as it self modifies its input argument "
-        "statement. Please see `_clear()` in `stmt.py`"
-    )
+def empty(expr, context):
+    if len(expr.args) != 1:
+        raise ParserException('function expects two parameters.', expr)
```

Comment:
```suggestion
        raise CompilerPanic('function expects one parameter.', expr)
```

Reply:
I think `ArgumentException` is the most fitting exception to raise here.  `ParserException` implies that the code can't be parsed. It also doesn't support annotation.